6238_prototype/src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -89,4 +99,26 @@ public void periodic() {
         SmartDashboard.putNumber("leftMotorsEncoderVelocity", talonLeftLeader.getSelectedSensorVelocity(PID_ID) * 0.1);
         SmartDashboard.putNumber("rightMotorsEncoderVelocity", talonRightLeader.getSelectedSensorVelocity(PID_ID) * 0.1);
     }
+
+    private double nativeUnitsToDistanceMeters(double sensorCounts){
+        double motorRotations = sensorCounts / kCountsPerRev;
+        double wheelRotations = motorRotations / kGearRatio;
+        double positionMeters = wheelRotations * (2 * Math.PI * Units.inchesToMeters(kWheelRadiusInches));
+        return positionMeters;
+    }
+
+    public void resetEncoders() {
+        talonLeftLeader.setSelectedSensorPosition(0);
+        talonRightLeader.setSelectedSensorPosition(0);
+    }
+
+    public double getPosition() {
+        return nativeUnitsToDistanceMeters(
+            talonLeftLeader.getSelectedSensorPosition() / 2
+            + talonRightFollowerTwo.getSelectedSensorPosition() / 2);
+    }
+
+    public double getAngle() {
+        return Math.IEEEremainder(ahrs.getAngle(), 360);
```

Comment:
Verify `getAngle` return degrees and not radians

Reply:
![image](https://user-images.githubusercontent.com/89051854/162096202-1b59ac55-2a8d-4674-b394-46589a0e6ee1.png)